src/main/java/com/example/chat/groupchatbackend/controllers/ConversationController.java

```diff
@@ -1,19 +1,16 @@
 package com.example.chat.groupchatbackend.controllers;
 
-import com.example.chat.groupchatbackend.Conversation;
-import com.example.chat.groupchatbackend.ConversationType;
-import com.example.chat.groupchatbackend.Message;
-import com.example.chat.groupchatbackend.User;
+import com.example.chat.groupchatbackend.*;
 import com.example.chat.groupchatbackend.repositories.ConversationsRepository;
 import com.example.chat.groupchatbackend.repositories.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
plase add the following line to the class `@CrossOrigin(origins = "*", allowCredentials = "true", maxAge = 3600L)`